packages/twenty-front/src/modules/object-record/record-merge/states/mergeSettingsState.ts

```diff
@@ -0,0 +1,9 @@
+import { MergeManySettings } from '@/object-record/hooks/useMergeManyRecords';
+import { createState } from 'twenty-ui/utilities';
+
+export const mergeSettingsState = createState<MergeManySettings>({
```

Comment:
question: what does a MergeManySetting represent? I'm a bit confuse by the meaning of priorityRecordIndex: 0

Reply:
naming could've been better I updated it to `conflictPriorityIndex` which is consistent with backend now